yt_dlp/extractor/funimation.py

```diff
@@ -6,24 +6,24 @@
 import string
 
 from .common import InfoExtractor
-from ..compat import compat_HTTPError
+from ..compat import compat_HTTPError, compat_str
```

Comment:
Now that the project is Python 3 only, isn’t `str` enough?

Reply:
yes, I use compat_str mostly out of habit